piglow/examples/strobe.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"time"
+
+	"github.com/goiot/devices/piglow"
+	"golang.org/x/exp/io/i2c"
+)
+
+func main() {
+	p, err := piglow.Open(&i2c.Devfs{Dev: "/dev/i2c-1", Addr: 0x54})
+	if err != nil {
+		panic(err)
+	}
+
+	defer p.Close()
+
+	if err := p.Reset(); err != nil {
+		panic(err)
+	}
+
+	for {
```

Comment:
A proper handling of a signal to quit and clear all states would be good. It does add a bit more complexity to the code example (see the oled96x96) but it's very useful in real life.